functions-and-scope.js

```diff
@@ -40,14 +64,45 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // * Hoe zorgt ik ervoor dat ik alle waardes uit de array kan langslopen, ook als de array wel 100 entries zou bevatten?
 // Log het antwoord in de terminal.
 
-// ---- Verwachte uitkomst: 6.642857142857143
+// ---Verwachte uitkomst: 6.642857142857143
+
+// * Gemiddelde berekenen: alle getallen bij elkaar optellen en vervolgens delen door hoeveel getallen het zijn
+// * verzamelen van de cijfers uit de array en dan door de array.length delen
+{
+let average=0;
+
+for (let i = 0; i < grades.length; i++) {
+    average = average + grades[i];
```

Comment:
Deze zou je dan zo kunnen doen:
```suggestion
    average += grades[i];
```